data/states/keno/main.py

```diff
@@ -49,9 +49,33 @@ def __init__(self, casino_player):
         self.label.rect.center = self.rect.center
         self.color = '#181818'
         self.casino_player = casino_player
-        self.bet = 0
-        self.is_paid = False
-        self.winnings = 0
+        self._bet = 0
+        self._is_paid = False
+        self._winnings = 0
+
+    @property
```

Comment:
It is highly discouraged to write setters and getters in python unless there is a reason to (the setter/getter does additional calculations).